components/brave_rewards/core/contribution/contribution.h

```diff
@@ -93,8 +94,29 @@ class Contribution {
 
   void OnNextMonthlyContributionTimeRead(absl::optional<base::Time> time);
 
-  void OnMonthlyContributionSet(base::OnceCallback<void(bool)> callback,
-                                bool success);
+  void OnMonthlyContributionSet(bool success);
+
+  struct ContributionRequest {
+    ContributionRequest(std::string publisher_id,
+                        double amount,
+                        bool set_monthly,
+                        base::OnceCallback<void(bool)> callback);
+
+    ContributionRequest(ContributionRequest&& other);
+
+    ~ContributionRequest();
```

Comment:
It's funny how `[chromium-style]` and special member function generation mess up this simple class.